svc/TrackService.js

```diff
@@ -10,8 +10,27 @@ import {stripTags} from 'hoist/utils/HtmlUtils';
 
 export class TrackService extends BaseService {
 
+    /**
+     * Create a Track Log entry.
+     * Client metadata is set automatically by the server's parsing of request headers.
+     *
+     * @param {Object|string} options - if a string, it will become the msg value.
```

Comment:
The multiple types get wrapped in parens `{(Object|string)}` (yeah...it's a lot of punctuation)

Reply:
fixed.